NBISweden/mimir
# feat: Add dev and staging environments

## Related issue(s) and PR(s)
This PR closes #2 and #3



## Type of change


## List of changes made

- This involves the use of docker compose profiles
- Submodules have been set-up for client and server

## Screenshot of the fix


## Testing

- Make sure the submodules are in your fs when cloning
- Run COMPOSE_PROFILES=dev docker compose up


## Definition of Done checklist
- [x] Make sure layout and text follows design sketches, if there are any
